chromium_src/components/bookmarks/browser/bookmark_model_unittest.cc

```diff
@@ -6,40 +6,59 @@
 #include "../../../../../components/bookmarks/browser/bookmark_model_unittest.cc"
 namespace bookmarks {
 
-TEST_F(BookmarkModelTest, BraveMigrateOtherNode) {
+TEST_F(BookmarkModelTest, BraveMigrateOtherNodeFolder) {
   // -- Bookmarks
   // |-- A
-  // -- Other Bookmarks
-  // |-- B
-  // |   |--B1.com
-  // |-- C.com
+  // |-- Other Bookmarks
+  //     |-- B
+  //     |   |--B1.com
+  //     |-- C.com
```

Comment:
there is no C.com bookmark, though this is not current PR change

Reply:
fixed